src/openforms/formio/tests/validation/test_map.py

```diff
@@ -4,6 +4,20 @@
 from .helpers import extract_error, validate_formio_data
 
 
+def _recursive_get_error_code(error):
```

Comment:
it should be possible to rewrite everything to be more explicit which makes this helper obsolete - see further remarks.

Test code should ideally be readable by absolute beginner developers that barely know to write an if/else statement and give them the feeling they understand what's going on. So, dead-simple!